### DISCUSSION TITLE
First time installing skins

### DISCUSSION THREAD

**Discussion by ImanolHP** [TT-ReBORN/Georgia-ReBORN#114] [Q&A] [answered]
I need help, after following all the steps I can't find any way to make the skin work
The dialog box to select columns ui does not appear

**Comment by TT-ReBORN** [comment, marked as answer]
1. [Download latest stable **foobar2000 32-bit** from here and save download to your desktop.](https://www.foobar2000.org/download)
2. Double click on foobar2000_v2.0.exe and click on the Next button, click on the I Agree button, select Portable installation and click on the Next button.
 Destination folder for installation should look like this: C:\Users\Administrator\Desktop\foobar2000, click on the Next button, click on the Install button.
Finally uncheck Run foobar2000 and click on the Finish button.
3. [Download Georgia-ReBORN v2.3.0 FINAL Beta 12e](https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/74#discussion-4190799) to your desktop and double click on the zip.
Drag and drop the profile folder to the installed portable foobar2000 folder.
Now click on the foobar2000 folder, click on the profile folder, click on the georgia-reborn folder, click on the fonts folder,
click on the Windows folder, select all fonts and right click and click on Install.
4. When everything installed, open the foobar2000 folder from your desktop and click foobar2000.exe.
